pkg/server/server.go

```diff
@@ -42,8 +42,10 @@ func NewServer(host string, port int, insecureSkipTLS bool, backends ...string)
 	// setting middleware before routes, otherwise it does not work (gin bug)
 	r.Use(ginzap.GinzapWithConfig(zapLogger, &ginzap.Config{}))
 	r.Use(ginzap.RecoveryWithZap(zapLogger, true))
-	r.GET("/", handleHealthz)
-	r.POST("/", sprayProxy.HandleProxy)
```

Comment:
This will break production. Please restore this.

Reply:
I will restore this once pr https://github.com/redhat-appstudio/sprayproxy/pull/30 get merged. For now I am leaving it as it is, so that others can test the behaviour.